src/qt/guiutil_font.h

```diff
@@ -18,12 +18,14 @@ namespace GUIUtil {
 enum class FontFamily : int {
     SystemDefault,
     Montserrat,
-    MaxVal = Montserrat
+    RobotoMono,
+    MaxVal = RobotoMono
 };
 
-static const std::array<std::pair<FontFamily, QString>, 2> AVAILABLE_FONTS{{
-    {FontFamily::SystemDefault, QString{"SystemDefault"}},
-    {FontFamily::Montserrat, QString{"Montserrat"}},
+static const std::array<std::tuple<FontFamily, QString, /*selectable=*/bool>, 3> AVAILABLE_FONTS{{
```

Comment:
so far as there is 3 different objects consider to define a tiny struct 

```
struct FontAvailability
{
  FontFamily family;
  QString family_name;
  bool selectable;
}
```


So there are no ugly comments such as `/*selectable=*/` all over codebase; and no magic std::get<1>(...) --- what is one here?